src/components/popover/popover.tsx

```diff
@@ -731,7 +731,14 @@ export class EuiPopover extends Component<Props, State> {
       let ariaDescribedby;
       let ariaLive: HTMLAttributes<any>['aria-live'];
 
-      if (ownFocus) {
+      const panelAriaModal = panelProps?.hasOwnProperty('aria-modal')
+        ? panelProps?.['aria-modal']
+        : 'true';
+      const panelRole = panelProps?.hasOwnProperty('role')
+        ? panelProps?.role
```

Comment:
Are the `?.`s necessary on the second line of the ternaries? I believe you should have already established that panelProps exists and has its own properties, so it can't be undefined or falsey

```suggestion
        ? panelProps.['aria-modal']
        : 'true';
      const panelRole = panelProps?.hasOwnProperty('role')
        ? panelProps.role
```